### PR TITLE
CLUSTERNATED

### PR DESCRIPTION
**Pull request by rafkhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,45 @@ $ npm run cover
 $ npm run start
 ```
 
+
+## Deploy to Clusternator
+
+#### Setup
+```
+#
+# SOMEHOW OBTAIN THIS REPO FIRST, WITHOUT GIT HISTORY.
```
Comment (SethDavenport) [inline README.md]: "How" is documented here: https://github.com/rangle/dev/blob/master/howtos/github/new-projects.md

Maybe we should link to that file in this one?

Comment (SethDavenport) [inline README.md]: Or add the directions to that file.

File: circle.yml [hunk 2]
```diff
@@ -1,3 +1,18 @@
 machine:
   node:
-    version: 4.1.0
\ No newline at end of file
+    version: 4.2.3
```
Comment (SethDavenport) [inline circle.yml]: Does circle ci support 4.2.3 now?

File: clusternator-scripts/clusternator-decrypt.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (SethDavenport) [inline clusternator-scripts/clusternator-decrypt.sh]: How much of this stuff is generated by clusternator init?

**Comment by rafkhan** [comment]
@SethDavenport I completely forgot to comment.
I need this to stay unmerged until the entire deploy cycle works. I still haven't setup master builds here. I'll address all of those tomorrow :)

**Comment by SethDavenport** [comment]
Summarizing recent discussion with the clusternator team:

We need to dockerize the starters, and make sure that projects generated from them can be clusternated.  But we shouldn't be checking in clusternator stuff to the starters themselves.

Closing this PR.

**Pull request closed without merging**